service/src/main/java/greencity/mapping/tariff/GetTariffInfoForEmployeeDtoMapper.java

```diff
@@ -0,0 +1,53 @@
+package greencity.mapping.tariff;
+
+import greencity.dto.LocationsDtos;
+import greencity.dto.RegionDto;
+import greencity.dto.courier.CourierTranslationDto;
+import greencity.dto.courier.GetReceivingStationDto;
+import greencity.dto.tariff.GetTariffInfoForEmployeeDto;
+import greencity.entity.order.TariffsInfo;
+import greencity.entity.user.Region;
+import org.modelmapper.AbstractConverter;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class GetTariffInfoForEmployeeDtoMapper extends AbstractConverter<TariffsInfo, GetTariffInfoForEmployeeDto> {
+    @Override
+    protected GetTariffInfoForEmployeeDto convert(TariffsInfo source) {
+        Region region = source.getTariffLocations() != null
+            ? source.getTariffLocations().iterator().next().getLocation().getRegion()
+            : null;
+        RegionDto regionDto = region != null ? RegionDto.builder().regionId(region.getId()).nameEn(region.getEnName())
+            .nameUk(region.getUkrName()).build() : null;
+
+        List<LocationsDtos> locationsDtos = source.getTariffLocations().stream()
```

Comment:
I see you check if `source.getTariffLocations()` https://github.com/ita-social-projects/GreenCityUBS/pull/955/files#diff-48d54bbc1999db3f0a3864df1665781366165526155d4d9d706ab791583c46dcR20
It means that we can have `NPE` here

Reply:
fixed